docs/edgeql/statements/delete.rst

```diff
@@ -10,18 +10,33 @@ DELETE
 
 .. eql:synopsis::
 
-    [ WITH <with-spec> [ , ... ] ]
-    DELETE <expr> ;
+    [ WITH <with-item> [, ...] ]
+
+    DELETE <expr>
+
+    [ FILTER <filter-expr> ]
+
+    [ ORDER BY <order-expr> [direction] [THEN ...] ]
+
+    [ OFFSET <offset-expr> ]
+
+    [ LIMIT  <limit-expr> ] ;
 
 :eql:synopsis:`WITH`
     Alias declarations.
 
     The ``WITH`` clause allows specifying module aliases as well
-    as expression aliases that can be referenced by the ``UPDATE``
+    as expression aliases that can be referenced by the ``DELETE``
     statement.  See :ref:`ref_eql_with` for more information.
 
 :eql:synopsis:`DELETE <expr>`
-    Remove objects returned by *expr* from the database.
+    In principle, the entire ``DELETE ...`` statement is syntactic
```

Comment:
```suggestion
    The :eql:synopsis:`DELETE <expr>` statement is syntactic sugar for 
```

Reply:
OK, I'll update the synopsis section to `DELETE ...` instead.